packages/flutter/lib/src/widgets/list_wheel_scroll_view.dart

```diff
@@ -388,8 +527,9 @@ class FixedExtentScrollPhysics extends ScrollPhysics {
 /// The children are rendered as if rotating on a wheel instead of scrolling on
 /// a plane.
 class ListWheelScrollView extends StatefulWidget {
-  /// Creates a box in which children are scrolled on a wheel.
-  const ListWheelScrollView({
+  /// Construct a list in which children are scrolled a wheel. Its children
```

Comment:
Here and in the other constructor's doc comment, I suggest keeping the same third-person verb tense as the original text (usually these verbs end with an 's'):

https://www.dartlang.org/guides/language/effective-dart/documentation#prefer-starting-function-or-method-comments-with-third-person-verbs

For instance:
`Constructs a list whose children are scrolled on a wheel`

Reply:
fixed.